Objects/call.c

```diff
@@ -1334,8 +1338,11 @@ _PyStack_UnpackDict(PyObject *const *args, Py_ssize_t nargs, PyObject *kwargs,
         return -1;
     }
 
-    /* Copy position arguments (borrowed references) */
-    memcpy(stack, args, nargs * sizeof(stack[0]));
+    /* Copy positional arguments */
+    for (i = 0; i < nargs; i++) {
+        Py_INCREF(args[i]);
+        stack[i] = args[i];
+    }
```

Comment:
Is there a reason to not keep `memcpy` for the bulk copy, and only add an INCREF loop?

Reply:
> Is there a reason to not keep `memcpy` for the bulk copy, and only add an INCREF loop?

Not really, it's mostly a matter of style. I like doing the `INCREF` at the same time as the assignment. I also find the `for` loop variant simpler to understand.